### ISSUE TITLE
Agda pragmas are rendered incorrectly

### ISSUE THREAD

**Issue by andreasabel** [kosmikus/lhs2tex#18]
A pragma like

  {-# NO_TERMINATION_CHECK #-}

is rendered in math mode, interpreting _ as subscript.  Correct would be a rendering in verbatim mode.

**Comment by patrick-nicodemus** [comment]
@andreasabel do you have a way to fix this, even a hacky workaround?

**Comment by andreasabel** [comment]
@patrick-nicodemus : @asr mentions a workaround above, have you tried it?

**Comment by patrick-nicodemus** [comment]
@andreasabel Yes, I tried it and I was unable to adapt it to my problem. I want to translate
```
\begin{code}
{-# BUILTIN EQUALITY _≡_ #-}
\end{code}
```

However I am unable to figure out how to escape the underscores around equality. I tried %format but I had no luck here. When I try 

```
%subst pragma a = "\mbox{{-#\verb|" a "| #-}}"
```
the code it generates is then passed as an argument to the `columns` macro, I think, or possibly to another macro  `ColumnHook`  (idk how to read this kind of LaTeX code) but either way it is apparently not possible to wrap something in a verbatim environment before passing it to a function - https://tex.stackexchange.com/questions/83893/verbatim-inside-a-command

And \texttt does not escape underscores.

**Comment by andreasabel** [comment]
@patrick-nicodemus You could try and fork lhs2tex and see whether you can implement what you want.  
Or use Agda's latex backend (which unfortunately lacks the convenience to format inline code).
